src/operations/transforms/rotate.test.js

```diff
@@ -0,0 +1,149 @@
+    [ new Float32Array([ -2, 12, 13 ]), new Float32Array([ -2, -18, 13 ]),
+      new Float32Array([ 8, -18, 13 ]), new Float32Array([ 8, 12, 13 ]) ],
+    [ new Float32Array([ -2, 12, -7 ]), new Float32Array([ -2, 12, 13 ]),
+      new Float32Array([ 8, 12, 13 ]), new Float32Array([ 8, 12, -7 ]) ],
+    [ new Float32Array([ -2, -18, -7 ]), new Float32Array([ 8, -18, -7 ]),
+      new Float32Array([ 8, -18, 13 ]), new Float32Array([ -2, -18, 13 ]) ]
+  ]
+  t.deepEqual(obs, exp)
+
+  rotated = rotateX(90, geometry)
+  obs = geom3.toPoints(rotated)
+  t.deepEqual(obs, exp)
+
+  // rotate about Y
+  rotated = rotate([0, -90, 0], geometry)
+  obs = geom3.toPoints(rotated)
+  exp = [
+    [ new Float32Array([ 12, -7, -2 ]), new Float32Array([ -18, -7, -2 ]),
+      new Float32Array([ -18, 13, -2 ]), new Float32Array([ 12, 13, -2 ]) ],
+    [ new Float32Array([ 12, -7, 8 ]), new Float32Array([ 12, 13, 8 ]),
+      new Float32Array([ -18, 13, 8 ]), new Float32Array([ -18, -7, 8 ]) ],
+    [ new Float32Array([ 12, -7, -2 ]), new Float32Array([ 12, -7, 8 ]),
+      new Float32Array([ -18, -7, 8 ]), new Float32Array([ -18, -7, -2 ]) ],
+    [ new Float32Array([ 12, 13, -2 ]), new Float32Array([ -18, 13, -2 ]),
+      new Float32Array([ -18, 13, 8 ]), new Float32Array([ 12, 13, 8 ]) ],
+    [ new Float32Array([ 12, -7, -2 ]), new Float32Array([ 12, 13, -2 ]),
+      new Float32Array([ 12, 13, 8 ]), new Float32Array([ 12, -7, 8 ]) ],
+    [ new Float32Array([ -18, -7, -2 ]), new Float32Array([ -18, -7, 8 ]),
+      new Float32Array([ -18, 13, 8 ]), new Float32Array([ -18, 13, -2 ]) ]
+  ]
+  t.deepEqual(obs, exp)
+
+  rotated = rotateY(-90, geometry)
+  obs = geom3.toPoints(rotated)
+  t.deepEqual(obs, exp)
+
+  // rotate about Z
+  rotated = rotate([0, 0, 180], geometry)
+  obs = geom3.toPoints(rotated)
+  exp = [
+    [ new Float32Array([ 2, 7, -12 ]), new Float32Array([ 2, 7, 18 ]),
+      new Float32Array([ 2, -13, 18 ]), new Float32Array([ 2, -13, -12 ]) ],
+    [ new Float32Array([ -8, 7, -12 ]), new Float32Array([ -8, -13, -12 ]),
+      new Float32Array([ -8, -13, 18 ]), new Float32Array([ -8, 7, 18 ]) ],
+    [ new Float32Array([ 2, 7, -12 ]), new Float32Array([ -8, 7, -12 ]),
+      new Float32Array([ -8, 7, 18 ]), new Float32Array([ 2, 7, 18 ]) ],
+    [ new Float32Array([ 2, -13, -12 ]), new Float32Array([ 2, -13, 18 ]),
+      new Float32Array([ -8, -13, 18 ]), new Float32Array([ -8, -13, -12 ]) ],
+    [ new Float32Array([ 2, 7, -12 ]), new Float32Array([ 2, -13, -12 ]),
+      new Float32Array([ -8, -13, -12 ]), new Float32Array([ -8, 7, -12 ]) ],
+    [ new Float32Array([ 2, 7, 18 ]), new Float32Array([ -8, 7, 18 ]),
+      new Float32Array([ -8, -13, 18 ]), new Float32Array([ 2, -13, 18 ]) ]
+  ]
+  t.deepEqual(obs, exp)
+
+  rotated = rotateZ(180, geometry)
+  obs = geom3.toPoints(rotated)
+  t.deepEqual(obs, exp)
+})
+
+test('rotate: rotating of multiple objects produces expected changes', t => {
```

Comment:
sorry but I still am not fond of the 'expected changes' formulation for tests: I do not know for example in this case what the expected should be: just apply the rotation to each object individually, or consider them as a group and use centroid of the group to rotate ?

Reply:
OK. I'll try to make the tests more explicit but just want to test the general functionality. In this case, I grouped a bunch of tests together.

Reply:
I meant only the title of the test, not the test itself :)

Reply:
:)
Updated the titles for mirror test suites. If those look better then I will update the others test suites.